common/cgroup/cgroup.go

```diff
@@ -206,29 +267,56 @@ func parseCgroupController(cgroupPath, controller string) ([]string, error) {
 	return nil, fmt.Errorf("controller %q not found", controller)
 }
 
-// GetOwnCgroupPath returns the cgroup path of this process in controller
+func parseProcCgroupInfo(procCgroupInfoPath string) (string, error) {
```

Comment:
This seems to be a v2 specific parser, right? If so, IMHO it should be mentioned somewhere.